core/src/commonMain/resources/MR/base/strings.xml

```diff
@@ -4,6 +4,12 @@
     <!--suppress CheckTagEmptyBody -->
     <string name="blank" translatable="false"></string>
     <string name="login">Login</string>
+    <string name="be_careful">"Be Careful"</string>
+    <string name="got_it">"Got it"</string>
+    <string name="learn_more">"Learn more"</string>
+    <string name="social_warning_one">"Avoid sharing personal information"</string>
+    <string name="social_warning_two">"Report any inappropriate behavior"</string>
```

Comment:
Just a small suggestion  — instead of numbering the warnings as one, two, three, maybe we could try using descriptive names? It might make things a bit clearer 
`<string name="social_warning_no_personal_info">Avoid sharing personal information</string>
`
`<string name="social_warning_report_behavior">Report any inappropriate behavior</string>
`
`<string name="social_warning_respect_others">Be respectful in interactions</string>
`